src/amo/components/FeedbackForm/index.js

```diff
@@ -82,7 +82,9 @@ export const CATEGORY_HATEFUL_VIOLENT_DECEPTIVE = 'hateful_violent_deceptive';
 // E
 export const CATEGORY_ILLEGAL = 'illegal';
 // F
-export const CATEGORY_OTHER = 'other';
+// Note that `other` category technically exists but only for add-ons, it's for
+// the previous abuse reporting mechanism.
+export const CATEGORY_OTHER = 'something_else';
```

Comment:
Nit 1: Can you rename the variable to CATEGORY_SOMETHING_ELSE.

This is used in 9 files and it might be a bit confusing to have the name not correspond to the value, especially when all of the other constants do correspond.

Nit 2: There does not seem to be any test which confirms this value is correct according to what the backend expects.
Is that okay?

Reply:
I'll rename it yeah.

We typically don't have tests for constants values used straight up like that, I think that's ok.